test/expected/issue_40_quoted_regtype.out

```diff
@@ -0,0 +1,86 @@
+select
+    seed_uuid(1),
+    'public.notes',
+    jsonb_build_object(
+        'role', 'authenticated',
+        'sub', seed_uuid(2)::text
+    ),
+    array[('primary_color', 'eq', 'RED')::realtime.user_defined_filter];
+insert into public.notes(id, primary_color)
+values
+    (1, 'RED'),   -- matches filter
+    (2, 'GREEN'); -- does not match filter
+select
+    rec,
+    is_rls_enabled,
+    subscription_ids,
+    errors
+from
+   walrus;
+                      rec                       | is_rls_enabled |            subscription_ids            | errors 
+------------------------------------------------+----------------+----------------------------------------+--------
+ {                                             +| f              | {f4539ebe-c779-5788-bbc1-2421ffaa8954} | {}
+     "type": "INSERT",                         +|                |                                        | 
+     "table": "notes",                         +|                |                                        | 
+     "record": {                               +|                |                                        | 
+         "id": 1,                              +|                |                                        | 
+         "primary_color": "RED"                +|                |                                        | 
+     },                                        +|                |                                        | 
+     "schema": "public",                       +|                |                                        | 
+     "columns": [                              +|                |                                        | 
+         {                                     +|                |                                        | 
+             "name": "id",                     +|                |                                        | 
+             "type": "int4"                    +|                |                                        | 
+         },                                    +|                |                                        | 
+         {                                     +|                |                                        | 
+             "name": "primary_color",          +|                |                                        | 
+             "type": "Color"                   +|                |                                        | 
+         }                                     +|                |                                        | 
+     ],                                        +|                |                                        | 
+     "commit_timestamp": "2000-01-01T00:01:01Z"+|                |                                        | 
+ }                                              |                |                                        | 
+ {                                             +| f              | {}                                     | {}
```

Comment:
@olirice Can you throw in a successful test when RLS is enabled? Just want to make sure it absolutely works in either security scenarios.